.ci/azure-pipelines.yml

```diff
@@ -117,30 +120,37 @@ jobs:
         path: '$(System.ArtifactsDirectory)'
         runVersion: 'latest'
 
-    - task: AndroidSigning@3
+    - task: DownloadSecureFile@1
+      displayName: 'Download KeyStore'
+      name: 'KeyStore'
+      inputs:
+        secureFile: 'keystore'
+
+    - script: '$(find / -name apksigner -print -quit) sign --ks ${KEYSTORE_SECUREFILEPATH} --ks-pass pass:${PASSWORD} ${SYSTEM_ARTIFACTSDIRECTORY}/jellyfin-android-${ARTIFACTSUFFIX}-${TAG}.apk'
       displayName: 'Sign APK'
       condition: eq(variables['BuildConfiguration'], 'release')
-      inputs:
-        apkFiles: '$(System.ArtifactsDirectory)/*.apk'
-        keystoreFile: keystore
-        keystorePassword: $(KeyStorePassword)
+      env:
+        PASSWORD: $(KeyStorePassword)
 
     - task: GithubRelease@0
       displayName: 'GitHub Upload'
       inputs:
         gitHubConnection: Jellyfin Release Download
         repositoryName: jellyfin/jellyfin-android
         assets: '$(System.ArtifactsDirectory)/*.apk'
+        action: 'edit'
+        assetUploadMode: 'replace'
```

Comment:
Does this replace all assets or only if there is a name collision?

Reply:
the latter